### PR TITLE
extend wait for assertion time.#675

### PR DESCRIPTION
**Pull request by btoutnp**
Please review #675.

### CODE REVIEW

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java [hunk 1]
```diff
@@ -48,6 +49,7 @@ public void test01_01_requestControllerHandling() {
 
         // WARN Level Log
         dbLogProvider.waitForAssertion();
+        assertTrue(driver.getTitle().contains("terasoluna-gfw-functionaltest"));
```
Comment (yoshikawaa) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java]: - [ ] It seems to use `asertThat`.

**Comment by btoutnp** [comment]
already changed to assertThat,please confirm.

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java [hunk 2]
```diff
@@ -69,6 +69,7 @@ public void test01_02_requestControllerHandling() {
 
         // INFO Level Log
         dbLogProvider.waitForAssertion();
+        assertThat(driver.getTitle(), is("terasoluna-gfw-functionaltest"));
         assertThat(dbLogProvider.countContainsMessageAndLevelsAndLogger(
                 "\\[w.xx.0002\\] ResultMessages \\[type=info, list=\\[ResultMessage \\[code=i.gt.me.0004, args=\\[\\], text=null\\]\\]\\]",
                 "INFO", "org.terasoluna.gfw.common.exception.ExceptionLogger"),
```
Comment (yoshikawaa) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java]: @btoutnp 
- [ ] The cause of the Travis error is that in some test cases the check at `assertThat(driver.getTitle()` is missing.
For all of test cases using DB log assertion you need to assert title.

For example, https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/2ae40165f3103002b06630c9f037d9f138a8cdd1/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java#L95-L96

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java [hunk 3]
```diff
@@ -48,6 +47,7 @@ public void test01_01_requestControllerHandling() {
 
         // WARN Level Log
         dbLogProvider.waitForAssertion();
```
Comment (itouyun) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java]: I think that `dbLogProvider.waitForAssertion();` is not necessary...

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java [hunk 4]
```diff
@@ -46,8 +45,7 @@ public ExceptionHandlingTest() {
     public void test01_01_requestControllerHandling() {
         driver.findElement(By.id("requestControllerHandling_01_01")).click();
 
-        // WARN Level Log
```
Comment (itouyun) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java]: line comments is deleted...
However I think that comments are unnecessary, too.

**Comment by itouyun** [comment]
@btoutnp 
Thank you for your work!
Pleaase backport all version!

**Pull request merged**